### PR TITLE
feat: add FAQ section

### PR DESCRIPTION
**Pull request by komal-agarwal5**
## Related Issue
#340 

## Description
I have added an FAQ section using accordions which displays the most frequently asked question answers.It will help clear the common queries of users.I have also added some sample question answers based on git-re from my own.

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
Screenshot 2024-07-03 190936

https://github.com/ashutosh-rath02/git-re/assets/122633300/7ee36541-222a-443f-96f3-7c3a767af08c

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
Thank you for submitting your pull request! 🙌 We'll review it as soon as possible. In the meantime, please ensure that your changes align with our [CONTRIBUTING.md](https://github.com/ashutosh-rath02/git-re/blob/main/CONTRIBUTING.md). If there are any specific instructions or feedback regarding your PR, we'll provide them here. Thanks again for your contribution! 😊

**Comment by ashutosh-rath02** [comment]
image

@komal-agarwal5 resolve this please and remove the underline please

**Comment by komal-agarwal5** [comment]
> image
> 
> @komal-agarwal5 resolve this please and remove the underline please

@ashutosh-rath02 I made the corrections, is it working now?

File: components/Faq.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"useClient";
```
Comment (ashutosh-rath02) [inline components/Faq.tsx:1]: use client

**Review by ashutosh-rath02 — CHANGES_REQUESTED** [review]

File: components/Faq.tsx (hunk 1 above)
Comment (ashutosh-rath02) [reply components/Faq.tsx:1]: Hey @komal-agarwal5 yes it's working now, please do this small change and we are gtg

Comment (komal-agarwal5) [reply components/Faq.tsx]: Done

**Review by ashutosh-rath02 — APPROVED** [review]
LGTM

**Comment by komal-agarwal5** [comment]
@ashutosh-rath02 sir can you please add the gssoc and level labels?

**Pull request merged**